### DISCUSSION TITLE
Cannot Open Idea Statica File using Grasshopper

### DISCUSSION THREAD

**Discussion by huan-tran** [idea-statica/ideastatica-public#710] [Rhino/Grasshopper]
I have 2 similar Idea Statica files but I can only open (by Grasshopper ) one file. Something wrong with the script.
I used the latest gh version.
![1](https://github.com/user-attachments/assets/32efce28-b328-48c3-9956-190f1e152ee4)
![2](https://github.com/user-attachments/assets/48c145de-6d04-4e5c-8c49-42ea4a5445e1)

**Comment by nathanlukeIDEA** [comment]
Hi @huan-tran, can you send me the file which is not working? nathan.luke@ideastatica.com. Also which is the latest version of IDEA StatiCa you are using?

**Comment by huan-tran** [comment]
hi Nathan,
Please check these files. I'm using Idea Statica version 24.0.5.1401. Gh version is 0.1.13.
[https://res.public.onecdn.static.microsoft/assets/mail/file-icon/png/generic_16x16.png]can open.ideaCon<https://arupapc-my.sharepoint.com/:u:/g/personal/huan_tran_arup_com/ER7qHGvHIUdJn6DO-OUq6jQBaqRMkMoJPj9fhAcwV66Gqw>
[https://res.public.onecdn.static.microsoft/assets/mail/file-icon/png/generic_16x16.png]cannot open.ideaCon<https://arupapc-my.sharepoint.com/:u:/g/personal/huan_tran_arup_com/EaUtVU32H-1Ir8BTd-CxriwB-Y6b0eLDmXV-zeod5BkK4Q>

Thank you.


Regards,



Huan Tran

Structural Engineer



Arup

17F E-town Central 11 Doan Van Bo District 4 HCMC Vietnam

t:  +848 6291 4062  f:  +848 6291 4072

www.arup.com/<http://www.arup.com/>



Follow Arup on LinkedIn<https://www.linkedin.com/company/arup> │ @ArupGroup<https://twitter.com/arupgroup>

Discuss the issues affecting the built environment at Arup Perspectives<https://www.arup.com/en/perspectives>



________________________________
